<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>        
    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Esha  Manchanda</title>
    <meta name="author" content="Esha  Manchanda" />
    <meta name="description" content="" />


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

    <!-- Styles -->
    <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22></text></svg>">
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://esha1701.github.io/">

    <!-- Dark Mode -->
    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
  </head>

  <!-- Body -->
  <body class="fixed-top-nav">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item active">
                <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a>
              </li>
              
              <!-- Blog -->
              <li class="nav-item ">
                <a class="nav-link" href="/blog/">blog</a>
              </li>

              <!-- Other pages -->
                  <!-- Teaching -->
         

              <!-- Toogle theme mode -->
              <div class="toggle-container">
                <a id="light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </a>
              </div>
            </ul>
          </div>
        </div>
      </nav>
    </header>

    <!-- Content -->
    <div class="container mt-5">
      <!-- about.html -->
      <div class="post">
        <header class="post-header">
          <h1 class="post-title">
           <span class="font-weight-bold">Esha</span>  Manchanda
          </h1>
          <p class="desc"><a href="#">Student Researcher</a></p>
        </header>

        <article>
          <div class="profile float-right">
<figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/prof_pic-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/prof_pic-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/prof_pic-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid z-dept-1 rounded" src="/assets/img/prof_pic.jpg" alt="prof_pic.jpg">

  </picture>

</figure>

          </div>

          <div class="clearfix">
            <p>Hi! I am Esha, a Masters in Biomedical Data Science student at  <a href="https://www.ntu.edu.sg" target="_blank" rel="noopener noreferrer">Nanyang Technological University</a>, Singapore. I graduated from <a href="https://www.ashoka.edu.in" target="_blank" rel="noopener noreferrer">Ashoka University</a> with a Bachelors in Computer Science. I am passionate about improving human health through the confluence of technology and personalised medicine. My research interests lie at the intersection of neuroscience, immunology, artificial intelligence and epigenetics. I am especially interested in reversing neurodegeneration and remyelination in multiple sclerosis.</p>

<p>I have leveraged computational modeling, data science and machine learning across various healthcare projects. At present, I am doing research at the <a href="https://www.lo-zeng-labs.com" target="_blank" rel="noopener noreferrer">Lo Lab</a> at LKC Medicine.

<p>Outside of academia I love playing badminton, running, solo travelling and learning korean!</p>


          </div>

              
   
      <br />     
              
              
           
          
          <!-- Selected Projects -->
          <div class="publications">
        <h2 style="font-weight: 400;">Ongoing Research</h2>
            <ol class="bibliography">
              






<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge"></abbr></div>
        <div class="row research-project" data-sort="2024-09-09">
            <div class="col-md-4">
                <!-- Image with hyperlink -->
                <a href="../assets/pdf/SfNPoster.pdf">
                    <img src="../assets/img/SfnPosterpic.png" alt="Hi" class="img-fluid">
                </a>
            </div>

        <!-- Entry bib key -->
        <div id="PhysRev.47.780" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Dysregulation of the Metabolic-Inflammatory Axis in Progressive Multiple Sclerosis</div>
          <!-- Author -->
          <div class="title"></div>

          <p class="text-muted">
                      <a href="#"><b>Esha Manchanda</b></a>,
                     <a href="#">Eka Saipuljumri</a>,
                      <a href="https://www.lo-zeng-labs.com">Jialiu Zeng</a>,
                      <a href="https://www.lo-zeng-labs.com">Chih Hung Lo</a>
                      <br />
                      <span style="font-style:italic;">In Preparation</span>&nbsp;
                      <br />
                       <i class="fas fa-medal"></i> Presented as a talk and poster at SBBI & Neuroscience Singapore 2024
                </p>
               <p> Metabolic impairments are shown to be implicated in the pathogenesis of neuroinflammatory diseases such as MS, but the exact mechanisms remain unknown.
                 In this study, we are investigated the role of mitochondrial, autophagic, and lysosomal dysfunction in progressive MS, exploring their interconnected roles in driving neuroinflammation and neurodegeneration. 
                        </p>
<!-- Links/Buttons -->
          <div class="info">
        <a href="../assets/pdf/SfN_Slides.pdf"><i class="fas fa-file-powerpoint" aria-hidden="true"></i> Slides</a>
          </div>

          
        </div>
      </div>
</li>


<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge"></abbr></div>
        <div class="row research-project" data-sort="2024-09-09">
            <div class="col-md-4">
                <!-- Image with hyperlink -->
                <a href="../assets/pdf/T1DPoster.pdf">
                    <img src="../assets/img/T1DPosterpic.png" alt="Hi" class="img-fluid">
                </a>
            </div>

        <!-- Entry bib key -->
        <div id="PhysRev.47.780" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">A LSTM Approach to Blood Glucose Predictions for Personalized T1D Management</div>
          <!-- Author -->
          <div class="title"></div>

          <p class="text-muted">
                      <a href="#"><b>Esha Manchanda</b></a>,
                      <a href="https://www.lo-zeng-labs.com">Jialiu Zeng</a>,
                      <a href="https://www.lo-zeng-labs.com">Chih Hung Lo</a>
                      <br />
                      <span style="font-style:italic;">In Preparation</span>&nbsp;
                      <br />
                       <i class="fas fa-medal"></i> Presented as a poster at FEBS-IUBMB-ENABLE 2024
                </p>
               <p> Type one diabetics need to be their own pancreas – it’s almost as though pumping blood to your body required having to know when it’s time to pump blood, rather than having your heart do it automatically. To ease the burden on T1Ds and improve blood glucose regulation, automated insulin delivery systems are being actively developed. Managing T1D, however, is significantly individualized— two people can have completely different blood glucose dynamics under identical exogenous conditions. Therefore, current systems are not well-suited to capture the unique disease patterns of a single individual. 
In this work, we have built a personalised blood glucose prediction model to address this inherent patient-patient variability.
</p>

          
        </div>
      </div>
</li>






<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge"></abbr></div>
        <div class="row research-project" data-sort="2024-09-09">
            <div class="col-md-4">
                <!-- Image with hyperlink -->
                <a href="../assets/pdf/4iPoster.pdf">
                    <img src="../assets/img/4iPosterpic.png" alt="Hi" class="img-fluid">
                </a>
            </div>

        <!-- Entry bib key -->
        <div id="PhysRev.47.780" class="col-sm-8">
        
          <!-- Title -->
          <div class="title">Multiplexed Imaging as an Innovative Approach for Single-Cell Spatial Profiling of Brain Pathology</div>
          <!-- Author -->
          <div class="title"></div>

          <p class="text-muted">
                      <a href="#">Eka Saipuljumri</a>,
                      <a href="#"><b>Esha Manchanda</b></a>,
                      <a href="https://www.lo-zeng-labs.com">Jialiu Zeng</a>,
                      <a href="https://www.lo-zeng-labs.com">Chih Hung Lo</a>
                      <br />
                      <span style="font-style:italic;">In Preparation</span>&nbsp;
                      <br />
                     
                </p>
          
               <p> 
                 An important aspect of understanding neuroinflammatory and neurodegenerative pathologies lies in defining cell populations and their spatial interactions, as well as the environmental cues that drive their specific expression profiles in specific regions of the brain. While single-cell RNA-seq is superior in isolated cells or nuclei compared to tissue with regard to gene coverage and sequencing depth, highly multiplexed protein imaging has a parametric depth of only tens to hundreds of markers but allows for precise outlining of cells with complex and irregular morphologies, which will improve the spatial analysis of individual cells. In this study, we present an innovative approach through serial immunofluorescence staining of tissue sections with repetitive cycles of immunolabeling, scanning, and antibody removal, termed iterative indirect immunofluorescent imaging (4i) for single-cell spatial profiling of brain pathologies. 
</p>

          
        </div>
      </div>
</li>

      
            </ol>        
             
           
          
          <!-- Selected Projects -->
          <div class="publications">
            <h2 style="font-weight: 400;">Past Work</h2>
            <ol class="bibliography">
              

              

<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge"></abbr></div>

        <!-- Entry bib key -->
        <div id="PhysRev.47.777" class="col-sm-8">
        
          <!-- Title -->
          <div class="title"> Undergraduate Thesis Candidate, Ashoka University</div>
          <!-- Author -->
          <div class="title"></div>


          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>August 2023 - May 2024</em>
          </div>
              
              
            <div class="abstract">
            <p> In my thesis, I explore the relationship between the Epstein-Barr Virus and Multiple Sclerosis. Through a mathematical modeling approach, my work studies the potential link between EBV and MS, closely examining the immune system's response to EBV infection. It investigates the role of T-cell exhaustion in uncontrolled EBV lytic activity in B cells, leading to chronic inflammation and potentially the development of MS.
            </p>
            </div>
<!-- Links/Buttons -->
          <div class="links">
            <a href="/assets/pdf/Thesis.pdf" class="btn btn-sm z-depth-0" role="button">Thesis</a>
          </div>


          
      </div>
</li>              
              
              
              
             












              


<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge"></abbr></div>

        <!-- Entry bib key -->
        <div id="PhysRev.47.777" class="col-sm-8">
        
          <!-- Title -->
          <div class="title"> Research Intern, Max Hospitals & Ashoka University</div>
          <!-- Author -->
          <div class="title"></div>


          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>January 2023 - present</em>
          </div>
              
              
              
            <div class="abstract">
            <p> 
              Over the past year, I have developed a growing interest in the gut microbiome and its impact on health. I worked on a project employing model-agnostic meta-learning techniques aimed at predicting diseases through the targeted analysis of fecal gut microbiome data. Beyond this, my interest extends to exploring the intricate connections between the gut microbiome and various health dimensions, including its role in both wellness and disease pathology.
            </p>
            </div>

<!-- Links/Buttons -->
          <div class="links">
            <a href="/assets/pdf/ML4HM.pdf" class="btn btn-sm z-depth-0" role="button">Report</a>
          </div>


          
      </div>
</li>              
              
              
              
             






              
              
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge"></abbr></div>

        <!-- Entry bib key -->
        <div id="PhysRev.47.777" class="col-sm-8">
        
          <!-- Title -->
          <div class="title"> Computational Genomics Research Intern, Ashoka University</div>
          <!-- Author -->
          <div class="title"></div>


          <!-- Journal/Book title and date -->
          <div class="periodical">
            <em>September 2022 - December 2022</em>
          </div>
              
              
              
            <div class="abstract">
            <p> 
              Traditional bulk methods (techniques used to analyze biological samples that contain a large population of cells) analyze an average over many cells, which can mask variations between individual cells. Single cell analysis helps investigate the heterogeneity between cells. Deciphering cell-cell heterogeneity at the chromatin level is very important while developing accurate disease models and determining patient responses to specific drugs. I conducted a comprehensive review of algorithms for single-cell analysis of accessible chromatin (scATAC-seq) to assess the computational challenges associated with scATAC-seq analysis including inherently sparse data, determination of features, peak calling, the impact of sequencing coverage and noise, and clustering performance.
            </p>
            </div>

          
      </div>
</li>              
              
              

   <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
          <h2 style="font-weight: 400;">Teaching Experience</h2>
            <p class="post-description"> I really enjoy teaching, and inspired by my many great teachers, I adopt a constructivist teaching style—where learners construct knowledge rather than just passively absorb information. </header>
 </p>
          </header>

      <article>
                <h3 id="teaching">Teaching Assistantship</h2>
                <ul>
                    <li>
                        <strong> Computational/Mathematical Biology (Spring 2024)</strong>, Ashoka University <br>
                        |<a href="https://sites.google.com/site/sudiptatung/">Professor Sudipta Tung</a> <br>
                    </li>
                   <li>
                        <strong> Introduction to Computer Programming (Monsoon 2021)</strong>, Ashoka University <br>
                     <a href="https://www.ashoka.edu.in/profile/subhashis-banerjee/">Professor Subhashis Banerjee</a> | <a href="../assets/pdf/ICPcontent.pdf">Course Content</a>
                    </li>
                   <li>
                         <strong> Computational/Mathematical Biology (Spring 2024)</strong>, Ashoka University <br>
                        |<a href="https://sites.google.com/site/sudiptatung/">Professor Sudipta Tung</a> |  <a href="https://sites.google.com/ashoka.edu.in/mathbio-monsoon-2021-btllz4c/home?authuser=0">Course Website</a>
                    </li>
                </ul>

         <h3 id="Workshops">Workshops</h2>
                <ul>
                    <li>
                        <strong> PySci Computational Biology Workshop (November 2023)</strong><br>
                        Organised by the IEEE Ashoka Student Student Branch [<a href="https://colab.research.google.com/drive/1cPBYgX1CtEnXMi8d2TaFXlIi-EYFqMut?usp=sharing">Hands-on exercise</a>]
                    </li>
                   
                    <li>
                        <strong> Tools for Web Scraping (October 2021)</strong><br>
                       Organised by the Computer Science Society [<a href="https://github.com/cs-ashoka/web-scraping-workshop">Code</a>]
                    </li>
                </ul>
               
            </article>

        </div>

    </div>


           
              
</ol>
          </div>

          <!-- Social -->
          <div class="social">
            <div class="contact-icons">
            <a href="mailto:esha004@ntu.edu.sg" title="email"><i class="fas fa-envelope"></i></a>
            <a href="https://github.com/esha1701" title="GitHub" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a>
            <a href="https://www.linkedin.com/in/eshamanchanda" title="LinkedIn" target="_blank" rel="noopener noreferrer"><i class="fab fa-linkedin"></i></a>
            

            </div>

            <div class="contact-note">
              Keep in touch!

            </div>
            
          </div>
        </article>

</div>
            

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2022 Esha  Manchanda. Powered by <a href="http://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Mansory & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/mansory.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
  <script src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script src="/assets/js/common.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
  </body>
</html>

